content/en/community/_index.md

```diff
@@ -24,7 +24,7 @@ Talk to us on our slack channels, which are part of the Kubernetes slack. Join K
 
 ### Office Hours
 
-We meet for office hours **every other Thursday** at 4pm UK / 11am US Eastern / 8am US Pacific.
+We meet for office hours **every other Thursday** at 16:00 UTC (*See your timezone [here](https://time.is/1500_in_UTC)*).
```

Comment:
just because it jumped out at me, but how come the url as 1500 instead of 1600? is that just how time.is works?

Reply:
Good spot, will fix.